README.md

```diff
@@ -7,7 +7,7 @@ A module for watching over scheduled processes: catching errors and sending mess
 
 ## Rationale
 
-supervisor provides a framework for scripts scheduled through Windows' Task Scheduler to report any errors and/or any logs via the handlers in `message_handlers.py`. Task Scheduler's built-in email tools require an email server running on the machine(?) and can't handle custom targets like Slack.
+supervisor provides a framework for scripts scheduled through Windows' Task Scheduler to send messages via the handlers in `messaging.py`. The messages can include gziped log files, progress reports, or execution summaries. The message handlers provide access to email, slack, and any other custom handler. These reporting methods are not supported by Task Scheduler. It also sends any uncaught exceptions via the registered message handlers.
```

Comment:
`message_handlers.py`, right?

Reply:
dah, gotta wake up today. I had some git conflicts so I just blew away my changes so I could pull and then tried to recreate.